### ISSUE TITLE
Feedback on miniex7 by Emma Ottilie

### ISSUE THREAD

**Issue by EmmaOttilie** [LurdKek/AP2017#12]
First of all; how dare you. This is so good. 
It looks so simple and yet it's really complex. And fun! :)

Your code is neat and well explained and I like that you put your object in a separate sketch, it makes it much more comprehensible. 
I know you had a hard time with this piece, so I just wanted to send kudos your way for actually accomplishing what you set out to do. 
My only note is that I would have loved a bit more text with your program. At first I wasn't sure how to control it (I'm so used to wasd) and I didn't understand what the slider did until I read your code. 
So maybe a tiny text on that, as well as your thoughts on objects and object oriented programs.
Otherwise I absolutely love this :)

So in conclusion; never speak to me again

**Comment by LurdKek** [comment]
:-*
